WebRoot/js/zimbraAdmin/globalconfig/view/GlobalConfigXFormView.js

```diff
@@ -329,6 +329,16 @@ GlobalConfigXFormView.updateRetentionPolicy = function(form) {
     }
 }
 
+GlobalConfigXFormView.sortRegisteredDevices = function(list) {
+    var sortOrder = ZaApp.getInstance().getCurrentController().getSortOrder();
+    if (sortOrder) {
+        list.sort(ZaItem.compareEmailAddress(true));
+    } else {
+        list.sort(ZaItem.compareEmailAddress(false));
+    }
```

Comment:
`list.sort(ZaItem.compareEmailAddress(typeof sortOrder !== 'undefined'));`

Reply:
here if we get sortorder as false then also it will send **true** value which is not correct

Reply:
can we do `list.sort(ZaItem.compareEmailAddress(Boolean(sortOrder) === true));`

Reply:
let me test that out 
how about list.sort(ZaItem.compareEmailAddress(sortOrder ? true : false ));

Reply:
sure if it works

Reply:
its working